corehq/apps/enterprise/views.py

```diff
@@ -66,8 +66,8 @@ def enterprise_dashboard(request, domain):
             EnterpriseReport.FORM_SUBMISSIONS,
         )],
         'current_page': {
-            'page_name': _('Enterprise Dashboard'),
-            'title': _('Enterprise Dashboard'),
+            'page_name': _('Enterprise Console'),
```

Comment:
please revert this change and the line below. we want to keep the page name the same "Enterprise Dashboard". just the section should change